src/common/timing/nav-timing.js

```diff
@@ -27,43 +27,60 @@ var DOM_CONTENT_LOAD_EVENT = 'domContentLoadedEvent'
 
 export var navTimingValues = []
 
-export function addPT (offset, pt, v = {}) {
+const PNT_TYPES = {
+  navigate: undefined,
+  reload: 1,
+  back_forward: 2,
+  prerender: 3
+}
+
+export function addPT (offset, pt, v = {}, isL1Api = false) {
   if (!pt) return
   v.of = offset
-  handleValue(0, v, 'n')
-  handleValue(pt[UNLOAD_EVENT + START], v, 'u')
-  handleValue(pt[REDIRECT + START], v, 'r')
-  handleValue(pt[UNLOAD_EVENT + END], v, 'ue')
-  handleValue(pt[REDIRECT + END], v, 're')
-  handleValue(pt['fetch' + START], v, 'f')
-  handleValue(pt[DOMAIN_LOOKUP + START], v, 'dn')
-  handleValue(pt[DOMAIN_LOOKUP + END], v, 'dne')
-  handleValue(pt['c' + ONNECT + START], v, 'c')
-  handleValue(pt['secureC' + ONNECT + 'ion' + START], v, 's')
-  handleValue(pt['c' + ONNECT + END], v, 'ce')
-  handleValue(pt[REQUEST + START], v, 'rq')
-  handleValue(pt[RESPONSE + START], v, 'rp')
-  handleValue(pt[RESPONSE + END], v, 'rpe')
-  handleValue(pt.domLoading, v, 'dl')
-  handleValue(pt.domInteractive, v, 'di')
-  handleValue(pt[DOM_CONTENT_LOAD_EVENT + START], v, 'ds')
-  handleValue(pt[DOM_CONTENT_LOAD_EVENT + END], v, 'de')
-  handleValue(pt.domComplete, v, 'dc')
-  handleValue(pt[LOAD_EVENT + START], v, 'l')
-  handleValue(pt[LOAD_EVENT + END], v, 'le')
+  handleValue(v.of, v, 'n', true)
+  handleValue(pt[UNLOAD_EVENT + START], v, 'u', isL1Api)
+  handleValue(pt[REDIRECT + START], v, 'r', isL1Api)
+  handleValue(pt[UNLOAD_EVENT + END], v, 'ue', isL1Api)
+  handleValue(pt[REDIRECT + END], v, 're', isL1Api)
+  handleValue(pt['fetch' + START], v, 'f', isL1Api)
+  handleValue(pt[DOMAIN_LOOKUP + START], v, 'dn', isL1Api)
+  handleValue(pt[DOMAIN_LOOKUP + END], v, 'dne', isL1Api)
+  handleValue(pt['c' + ONNECT + START], v, 'c', isL1Api)
+  handleValue(pt['secureC' + ONNECT + 'ion' + START], v, 's', isL1Api)
+  handleValue(pt['c' + ONNECT + END], v, 'ce', isL1Api)
+  handleValue(pt[REQUEST + START], v, 'rq', isL1Api)
+  handleValue(pt[RESPONSE + START], v, 'rp', isL1Api)
+  handleValue(pt[RESPONSE + END], v, 'rpe', isL1Api)
+  handleValue(pt.domLoading, v, 'dl', isL1Api)
+  handleValue(pt.domInteractive, v, 'di', isL1Api)
+  handleValue(pt[DOM_CONTENT_LOAD_EVENT + START], v, 'ds', isL1Api)
+  handleValue(pt[DOM_CONTENT_LOAD_EVENT + END], v, 'de', isL1Api)
+  handleValue(pt.domComplete, v, 'dc', isL1Api)
+  handleValue(pt[LOAD_EVENT + START], v, 'l', isL1Api)
+  handleValue(pt[LOAD_EVENT + END], v, 'le', isL1Api)
   return v
 }
 
 // Add Performance Navigation values to the given object
 export function addPN (pn, v) {
-  handleValue(pn.type, v, 'ty')
+  handleValue(PNT_TYPES[pn.type], v, 'ty')
```

Comment:
The L1 timing which have numeric `.type` would get undefined here instead of passing through